deploifai/project/info.py

```diff
@@ -0,0 +1,70 @@
+import click
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import (
+    pass_deploifai_context_obj,
+    DeploifaiContextObj,
+    is_authenticated, project_found,
+)
+
+
+@click.command()
+@pass_deploifai_context_obj
+@is_authenticated
+@project_found
+def info(context: DeploifaiContextObj):
+    """
+    Command to get the information of a project
```

Comment:
Simplify to `Get information about a project`

Reply:
refactored